tests/examples/test1/output/expected/all_cuda-115_arch-arm64.yaml

```diff
@@ -0,0 +1,16 @@
+# This file is generated by `rapids-dependency-file-generator`.
```

Comment:
Do we have (or want to add) tests that the generated conda environments are solvable / usable?

Reply:
Eh. That seems kind of difficult and potentially outside the scope of the tool. I think we can skip this for now.